homeassistant/components/onvif/camera.py

```diff
@@ -228,6 +229,16 @@ def perform_ptz(self, pan, tilt, zoom):
         finally:
             await stream.close()
 
+    @property
+    def supported_features(self):
+        """Return supported features."""
+        return SUPPORT_STREAM
+
+    @property
+    def stream_source(self):
+        """Return the stream source."""
+        return self._input.split(' ')[-1]
```

Comment:
The split is not needed here since the ONVIF input is already a single RTSP URL and is obtained directly from the ONVIF API.

Reply:
As discussed, please make the changes as this is a bit over my head! If you decide it's easier to close this PR and re-open that's fine too!

Thanks for your help!